### DISCUSSION TITLE
How to close this hand

### DISCUSSION THREAD

**Discussion by shenhaoguangDayDayUp** [google/model-viewer#3541] [Q&A]
### Description

How to close this hand?
<img width="299" alt="image" src="https://user-images.githubusercontent.com/17617821/174477219-1f35e37a-f4ad-4ac2-ae1e-cc9eb876ac87.png">

#### Live Demo

https://glitch.com/edit/#!/model-viewer

### Version

- model-viewer: vX.X.X

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
Not sure what you mean; it'll go away if you interact. You can also customize it: https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-attributes-interactionPrompt and https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-slots-interactionPrompt
